core/drivers/crypto/hisilicon/include/sec_main.h

```diff
@@ -0,0 +1,376 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/* Copyright (c) 2022-2024 HiSilicon Limited. */
+#ifndef __SEC_MAIN_H
+#define __SEC_MAIN_H
+
+#include "hisi_qm.h"
+
+#include <initcall.h>
+/*
+ * Version number maintenance rule:
+ * first.second.third: year.month.change num
+ */
+#define SEC_MODULE_VERSION "23.1.2"
```

Comment:
a tabulation before `"23.1.2"` is welcome, for consistency with the other macro definitions below.